src/helpers.test.js

```diff
@@ -312,6 +338,36 @@ describe('getPrettierOptions', () => {
   });
 });
 
+describe('getPrettierCliOptions', () => {
+  test('translates empty options', () => {
+    const expected = '';
+    const actual = getPrettierCliOptions({});
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('translates a single string option', () => {
+    const expected = '--print-width 80';
+    const actual = getPrettierCliOptions({ printWidth: 80 });
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('translates a single boolean option', () => {
+    const expected = '--single-quote true';
+    const actual = getPrettierCliOptions({ singleQuote: true });
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('translates complex options', () => {
+    const expected = '--single-quote true --print-width 100 --trailing-comma es5';
+    const actual = getPrettierCliOptions({ singleQuote: true, printWidth: 100, trailingComma: 'es5' });
```

Comment:
this is nit-picky, but can we reverse the order of `expected` and `actual` to match above?